documentation/asciidoc/computers/configuration/configuring-networking.adoc

```diff
@@ -1,183 +1,179 @@
+Your device will always try to connect to the in-range network with the highest non-negative priority value. You can also assign a network a negative priority; your device will only attempt to connect to a negative priority network if no other known network is in range. For example, consider three networks:
 
-By default, Raspberry Pi OS attempts to automatically configure all network interfaces by DHCP, falling back to automatic private addresses in the range 169.254.0.0/16 if DHCP fails. This is consistent with the behaviour of other Linux variants and of Microsoft Windows.
+----
+AUTOCONNECT-PRIORITY  NAME
+-1                    snake
+0                     rabbit
+1                     cat
+1000                  dog
+----
 
-=== Static IP Addresses
+- If all of these networks were in range, your device would first attempt to connect to the "dog" network.
+- If connection to the "dog" network fails, your device would attempt to connect to the "cat" network.
+- If connection to the "cat" network fails, your device would attempt to connect to the "rabbit" network.
+- If connection to the "rabbit" network fails, and your device detects no other known networks, your device will attempt to connect to the "snake" network.
 
-WARNING: If allocation of IP addresses is normally handled by a DHCP server on your network, allocating your Raspberry Pi a static IP address may cause an address conflict which may lead to networking problems.
+=== Configure DHCP
 
-If you want to allocate a static IP address to your Raspberry Pi, the best way to do so is to reserve an address for it on your router. That way your Raspberry Pi will continue to have its address allocated via DHCP but will receive the same address each time. A "fixed" address can be allocated by your DHCP server associating it with the MAC address of your Raspberry Pi. Management of IP addresses will remain with the DHCP server and this will avoid address conflicts and potential network problems.
+By default, Raspberry Pi OS attempts to automatically configure all network interfaces by DHCP, falling back to automatic private addresses in the range 169.254.0.0/16 if DHCP fails.
 
-However, if you wish to disable automatic configuration for an interface, and instead configure it statically, you can do so in `/etc/dhcpcd.conf`. For example:
+=== Assign a Static IP Address
 
-----
-interface eth0
-static ip_address=192.168.0.4/24	
-static routers=192.168.0.254
-static domain_name_servers=192.168.0.254 8.8.8.8
-----
+To allocate a static IP address to your Raspberry Pi, reserve an address for it on your router. That way your Raspberry Pi will continue to have its address allocated via DHCP but will receive the same address each time. A "fixed" address can be allocated by associating the MAC address of your Raspberry Pi with a static IP address in your DHCP server.
```

Comment:
What if a user has disabled DHCP on their router (or they're setting up a private LAN without a router) and wants to manually specify static IP addresses for all their devices? Or is that a configuration that we're explicitly not supporting / documenting ?

Reply:
> What if a user has disabled DHCP on their router (or they're setting up a private LAN without a router) and wants to manually specify static IP addresses for all their devices? Or is that a configuration that we're explicitly not supporting / documenting ?

The the user is an idiot, or an expert, and either way we can't help them.